playbooks/cluster/openshift/config.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- hosts: all
+- hosts: nodes
```

Comment:
Should be nodes and masters (they are part of the `OSEv3` group)

Reply:
all masters are nodes

Reply:
I believe, it is not correct assumption, you can have separate master without being in `nodes` hostgroup. Please correct me if I am wrong.

Reply:
AFAIK, all masters are in the nodes group so they can share the SDN but I can't find the original source. The table from the advanced install puts the master(s) in the `nodes` group in all scenarios though.

https://docs.openshift.com/container-platform/3.9/install_config/install/advanced_install.html#adv-install-example-inventory-files